domino_data/data_sources.py

```diff
@@ -476,6 +476,7 @@ def upload_fileobj(self, object_key: str, fileobj: Any) -> None:
     DatasourceDtoDataSourceType.S3CONFIG: ObjectStoreDatasource,
     DatasourceDtoDataSourceType.SNOWFLAKECONFIG: QueryDatasource,
     DatasourceDtoDataSourceType.SQLSERVERCONFIG: QueryDatasource,
+    DatasourceDtoDataSourceType.TABULARS3GLUECONFIG: QueryDatasource,
```

Comment:
This will need to wait for https://github.com/dominodatalab/domino-data/pull/52 to be merged